Sources/DockProgress/DockProgress.swift

```diff
@@ -97,4 +100,36 @@ public final class DockProgress {
 		progressCircle.progress = progressValue
 		progressCircle.render(in: cgContext)
 	}
+	
+	private static func drawProgressBadge(_ dstRect: CGRect, badgeLabel: String, color: NSColor) {
+		guard let cgContext = NSGraphicsContext.current?.cgContext else {
+			return
+		}
+
+		let radius = dstRect.width / 4.5
+		let newCenter = CGPoint(x: dstRect.maxX - radius - 4, y: dstRect.minY + radius + 4)
+
+		// Background
+		let badge = ProgressCircleShapeLayer(radius: Double(radius), center: newCenter)
+		badge.fillColor = CGColor(red: 0.94, green: 0.96, blue: 1, alpha: 1)
+
+		// Progress circle
+		let lineWidth: CGFloat = 6
+		let progressCircle = ProgressCircleShapeLayer(radius: Double(radius) - Double(lineWidth / 2), center: newCenter)
```

Comment:
Can you move `Double(radius) - Double(lineWidth / 2)` into a variable?

Reply:
I'll look into it 🙂